pytorch_lightning/plugins/training_type/single_tpu.py

```diff
@@ -82,7 +82,7 @@ def save_checkpoint(self, checkpoint: Dict[str, Any], filepath: _PATH) -> None:
             checkpoint: dict containing model and trainer state
             filepath: write-target file's path
         """
-        return self.checkpoint_io.save_checkpoint(checkpoint, filepath)
+        self.checkpoint_io.save_checkpoint(checkpoint, filepath)
```

Comment:
Can't we remove this override? Isn't global rank always 0 for single tpu?

Reply:
it was overridden because of my answer below.

Reply:
I think you can drop the override

Reply:
Done